installer/kali/releases

```diff
@@ -1,2 +1,3 @@
 kali*
 sana
```

Comment:
I wasn't sure about that as it's still supported. I will add a commit disabling sana.